consensus/src/dag/dag_fetcher.rs

```diff
@@ -103,17 +116,61 @@ impl DagFetcher {
                 .and_then(FetchResponse::try_from)
                 .and_then(|response| response.verify(&remote_request, &self.epoch_state.verifier))
             {
+                let ceritified_nodes = response.certified_nodes();
+                if ceritified_nodes.len() != remote_request.missing_count() {
```

Comment:
this should be written in the verify function above

Reply:
Probably need to also pass the bitmask and verify that the return certificates match the bitmask.

Reply:
the bitmask is there, the verify function takes in the request itself